modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -536,19 +594,24 @@ return view.extend({
 			uci.unset('dhcp', section, 'dns');
 		};
 
-		so = ss.option(form.Value, 'mac', _('MAC address'));
-		so.datatype = 'list(macaddr)';
+		var macwildcard = function(multicast) {
+			var m = this.value.match(/^([a-fA-F0-9]{2}|\*):([a-fA-F0-9]{2}:|\*:){4}(?:[a-fA-F0-9]{2}|\*)$/);
+			return this.assert(m != null && !(+m[1] & 1) == !multicast,
+				multicast ? _('valid multicast MAC address with wildcard (*)') : _('valid MAC address with wildcard (*)'));
+		}
+
+		so = ss.option(form.Value, 'mac',
+			_('MAC address(es)'),
+			_('The hardware address(es) of this entry/host, separated by spaces.'));
+		so.datatype = 'list(macwildcard)';
```

Comment:
This won't work. It needs to be something like `so.validate = function(section_id, value) { ... }`

Reply:
Ah yes, thank you, the cache probably hid this between test deploys. Fixed as per recommendations.